portal/src/locale-data/en.json

```diff
@@ -586,10 +586,23 @@
   "InviteAdminScreen.duplicated-error": "Invitation has already been sent",
 
   "AcceptAdminInvitationScreen.title": "Admin Invitation",
+  "AcceptAdminInvitationScreen.continue-to-authgear.label": "Continue to Authgear",
+  "AcceptAdminInvitationScreen.login.label": "Login",
+  "AcceptAdminInvitationScreen.login-with-another-user.label": "Login With Another User",
```

Comment:
Sorry I made an error writing this copy, it should be "Login **as** another user"